### PR TITLE
[bitnami/postgresql] Add improvements to comply with STIG

### PR DESCRIPTION
**Pull request by javsalgar**
**Description of the change**

It adds the following support 

* Enable the pgAudit module by default
* Log "login" and "logout" operations: 
* Provide more details in the log entries
* Set client_min_messages to a lower level 
* Allow configuring log timezone
* Allow configuring max_connections
* Allow removing `local` and `md5` entries in pg_hba.conf
* Allow configuring keepalive settings

**Benefits**

STIG Rules: V-72925, V-73069, V-73005, V-72961, V-73033, V-72933,  V-72929, V-72843, V-73067, V-73065, V-72919, V-72983, V-72903, V-73001, V-72973, V-72971, V-73025, V-72965, V-72963, V-72953, V-72955, V-72957, V-72959, V-72949, V-72947, V-72931,  V-72863, V-72863,  V-72863, V-73037, V-72849

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/postgresql/values.yaml [hunk 1]
```diff
@@ -237,6 +246,72 @@ masterAsStandBy:
...
+postgresqlTcpKeepalivesCount:
+
+## Statement timeout
+##
+postgresqlStatementTimeout:
+
+## Remove pg_hba.conf lines with the following comma-separated patterns
+## (cannot be used with custom pg_hba.conf)
+##
+postgresqlPghbaRemoveFilters:
```
Comment (miguelaeh) [inline bitnami/postgresql/values.yaml:313]: It seems that there is one more value here than used in the env vars. I can count 17 new values but only 16 new env vars.

**Review by miguelaeh — COMMENTED** [review]
Thank you very much for the PR!

I have some doubts, please take a look at my comments

File: bitnami/postgresql/values.yaml [hunk 2]
```diff
@@ -613,3 +701,7 @@ metrics:
     timeoutSeconds: 5
     failureThreshold: 6
     successThreshold: 1
+
+## Array with extra yaml to deploy with the chart. Evaluated as a template
+##
+extraDeploy: []
```
[the remark this answers is not in the document]
Comment (javsalgar) [reply bitnami/postgresql/values.yaml:707]: extra-list.yaml

**Review by miguelaeh — APPROVED** [review]
LGTM!!

**Pull request merged**